src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,24 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        contextMenuWasEnabled = atom.config.get('core.showAtomInShellContextMenu')
```

Comment:
Hello there. Any thoughts on this? Does it make sense to remove access `atom.config` and implement it this way?